### DISCUSSION TITLE
LFX Mentorship - Otel Logs Bridge

### DISCUSSION THREAD

**Discussion by yuriolisa** [open-telemetry/opentelemetry-rust#1520] [General]
Hi Team,

We are applying for the [LFX-Mentorship](https://github.com/open-telemetry/community/issues/1894) program and one of the ideas to suggest as a project to the mentees would be implementing the OpenTelemetry Logs Bridge API specification as defined through the following link: https://opentelemetry.io/docs/specs/otel/logs/bridge-api/
With that being said, could we define the Rust repository as a potential OpenTelemetry project to receive pull requests from the Mentorship program?

cc: @jpkrohling 

Yuri

**Comment by lalitb** [comment]
@yuriolisa OpenTelemetry Log Bridge API is already implemented for OpenTelemetry Rust. But in general, it should be fine to receive pull requests from the Mentorship program.
